docs/logging.rst

```diff
@@ -135,16 +135,21 @@ of the Astropy configuration file::
     log_exceptions = False
 
     # Whether to always log messages to a log file
-    log_to_file = True
+    log_to_file = False
 
-    # The file to log messages to
-    log_file_path = '~/.astropy/astropy.log'
+    # The file to log messages to. When '', it defaults to a file `astropy.log` in
+    # the astropy config directory.
+    log_file_path = ''
 
     # Threshold for logging messages to log_file_path
     log_file_level = 'INFO'
 
     # Format for log file entries
-    log_file_format = '%(asctime)s, %(origin)s, %(levelname)s, %(message)s'
+    log_file_format = '%(asctime)r, %(origin)r, %(levelname)r, %(message)r'
```

Comment:
Changing this does not seem necessary. I think this example is to show a non-default setting.